persistence/src/commonMain/sqldelight/com/wire/kalium/persistence/db/Teams.sq

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE Team (
+    id TEXT NOT NULL PRIMARY KEY,
+    name TEXT
+);
+
+insertTeam:
+INSERT INTO Team(id, name)
```

Comment:
Since the slow sync can be performed many times we need to handle the case when a team already exists in the database and we need to update it instead. You can look at Users.sq for how we solve this using ON CONFLICT.

Reply:
added `ON CONFLICT(id) DO UPDATE SET name = excluded.name;` 👌🏻 nice pointing it out!